### ISSUE TITLE
Term Request: Corneal endothelial pigment deposits

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11762]
**Preferred term label:**
Corneal endothelial pigment deposits

**Synonyms:**
* Krukenberg spindle
* Krukenberg's spindle
* Pigment deposits on corneal endothelium
* Endothelial pigment dusting
* Corneal endothelial pigmentation
* Pigmented keratic deposits

**Definition (free text):**
The presence of melanin pigment granules deposited on the posterior corneal surface (corneal endothelium). These deposits often aggregate along aqueous convection currents in a vertical, spindle-shaped pattern (Krukenberg spindle) or appear as diffuse pigment dusting, and are characteristically observed in pigment dispersion syndrome, pigmentary glaucoma, anterior uveitis, pseudoexfoliation syndrome, or following ocular trauma.

**Parent term (use hpo.jax.org/app):**
Abnormal corneal endothelium morphology (HP:0011488)

**Your nano-attribution (ORCID):**
0000-0001-9042-0508
